src/components/form/CreateProjectTemplateForm.vue

```diff
@@ -1,28 +1,51 @@
 <template>
   <q-form
-    class="q-gutter-md create-project-template-form"
+    class="create-project-template-form"
     data-cy="create-project-template-form"
     @submit="onSubmit"
   >
-    <q-input
-      v-model="projectName"
-      :label="$t('page.home.project.name')"
-      :rules="[
-        v => notEmpty($t, v),
-        v => isUniqueProjectName($t, projectNames, v)
-      ]"
-      filled
-      lazy-rules
-      data-cy="name-input"
-    />
+    <div class="row q-gutter-md">
+      <div class="flex-2">
+        <div class="text-subtitle2">
+          Rename this template for your project
+        </div>
+        <q-input
+          v-model="projectName"
+          :label="$t('page.home.project.name')"
+          :rules="[
+            v => notEmpty($t, v),
+            v => isUniqueProjectName($t, projectNames, v)
+          ]"
+          outlined
+          lazy-rules
+          data-cy="name-input"
+        />
+        <div class="text-subtitle2 q-mb-md">
+          Content
+        </div>
+        <q-input
+          v-model="templateDescription"
+          filled
+          data-cy="content-input"
+          disable
```

Comment:
no linter error ? data-cy must be last (if no methods/events)

Reply:
done ad no linter error